editor/code/package.json

```diff
@@ -144,12 +144,12 @@
     ]
   },
   "devDependencies": {
-    "@types/node": "^10.14.15",
+    "@types/node": "^18.11.18",
     "@types/throttle-debounce": "^5.0.0",
-    "@types/vscode": "^1.73.0",
-    "esbuild": "^0.15.14",
+    "@types/vscode": "^1.74.0",
```

Comment:
For now our base is 1.73, do we need 1.74 for some feature? Otherwise let's stay a 1.73.

Rest of changes are ok to me.